crypto/async/async.c

```diff
@@ -240,11 +252,17 @@ int ASYNC_start_job(ASYNC_JOB **job, ASYNC_WAIT_CTX *wctx, int *ret,
 
         ctx->currjob->func = func;
         ctx->currjob->waitctx = wctx;
+        libctx = get_default_context();
```

Comment:
There's already a non-static function  you can use:

``` C
        libctx = openssl_ctx_get_concrete(NULL);
```

I would rather not open up `get_default_context`

Reply:
Oh... Right, my PR, I can fix as I please :wink:

Reply:
Right!

Reply:
Done